var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -86,15 +87,16 @@ class Hdf5(CMakePackage):
             values=('default', 'v114', 'v112', 'v110', 'v18', 'v16'),
             multi=False)
 
-    depends_on('cmake@3.12:', type='build')
-
-    depends_on('mpi', when='+mpi')
+    if sys.platform != 'win32':
+        depends_on('mpi', when='+mpi')
```

Comment:
Since this PR includes an MPI implementation for Windows, can we allow depends_on('mpi', when=+mpi) for Windows as well? If not (i.e. if this PR will be merged without that) then this should add conflicts('+mpi', when='platform=windows'), and this if can also be removed in that case).

Reply:
I think to be on the safe side I'll make this a conflicts statement and then see how David Alexander's PR goes along. Will be in next commit